client/build/contracts/CappedToken.json

```diff
@@ -0,0 +1,1752 @@
+{
```

Comment:
Since you're using the build folder, you should remove the present .gitignores under the contracts' packages.

Reply:
and you should include that folder to be ignored in the main gitignore - https://github.com/kowala-tech/kcoin/blob/dev/.gitignore.